src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -1956,6 +1956,8 @@ public final int getPriority() {
      * @param      name   the new name for this thread.
      * @throws     SecurityException  if the current thread cannot modify this
      *             thread.
+     *
+     * @spec jni/index.html Java Native Interface Specification
```

Comment:
The link to the JNI spec in this method is from implNote so I'm wondering if the spec link is needed here.

Reply:
Leave it in for now and we can look at it again when the external spec page is in place. My comment is mostly that it will look a bit strange to link to this method because it's text in an implNote rather than spec.